gbpservice/nfp/configurator/agents/loadbalancer_v2.py

```diff
@@ -0,0 +1,979 @@
+                                             provisioning_status,
+                                         'operating_status':
+                                             operating_status,
+                                         obj_type: obj}}]
+               }
+        self.notify._notification(msg)
+
+    # TODO(jiahao): copy from v1 agent need to review
+    def update_pool_stats(self, pool_id, stats, context, pool=None):
+        """ Enqueues the response from LBaaS operation to neutron plugin.
+
+        :param pool_id: pool id
+        :param stats: statistics of that pool
+
+        """
+        msg = {'info': {'service_type': lb_constants.SERVICE_TYPE,
+                        'context': context.to_dict()},
+               'notification': [{'resource': 'pool',
+                                 'data': {'pool_id': pool_id,
+                                          'stats': stats,
+                                          'notification_type': (
+                                                        'update_pool_stats'),
+                                          'pool': pool_id}}]
+               }
+        self.notify._notification(msg)
+
+    # TODO(jiahao): copy from v1 agent need to review
+    def vip_deleted(self, vip, status, agent_info):
+        """ Enqueues the response from LBaaS operation to neutron plugin.
+
+        :param vip: object type
+        :param vip_id: object id
+        :param status: status of the object to be set
+
+        """
+        msg = {'info': {'service_type': lb_constants.SERVICE_TYPE,
+                        'context': agent_info['context']},
+               'notification': [{'resource': agent_info['resource'],
+                                 'data': {'vip_id': vip['id'],
+                                          'vip': vip,
+                                          'notification_type': 'vip_deleted',
+                                          'status': status}}]
+               }
+        self.notify._notification(msg)
+
+
+"""Implements APIs invoked by configurator for processing RPC messages.
+
+RPC client of configurator module receives RPC messages from REST server
+and invokes the API of this class. The instance of this class is registered
+with configurator module using register_service_agent API. Configurator module
+identifies the service agent object based on service type and invokes ones of
+the methods of this class to configure the device.
+
+"""
+
+
+class LBaaSv2RpcManager(agent_base.AgentBaseRPCManager):
+    def __init__(self, sc, conf):
+        """Instantiates child and parent class objects.
+
+        :param sc: Service Controller object that is used for interfacing
+        with core service controller.
+        :param conf: Configuration object that is used for configuration
+        parameter access.
+
+        """
+
+        super(LBaaSv2RpcManager, self).__init__(sc, conf)
+
+    def _send_event(self, event_id, data, serialize=False, binding_key=None,
```

Comment:
see [previous](https://github.com/oneconvergence/group-based-policy/pull/358#discussion_r65623635) comment for generalization.